connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -26,11 +26,14 @@ import org.apache.spark.sql.connect.planner.DataTypeProtoConverter
 
 /**
  * A collection of implicit conversions that create a DSL for constructing connect protos.
+ *
+ * All classes in connect/dsl are considered an internal API to Spark Connect and are subject to
+ * change between minor releases.
  */
 
 package object dsl {
 
-  object expressions { // scalastyle:ignore
+  private[connect] object expressions { // scalastyle:ignore
```

Comment:
If we document this as an internal package, I think it's fine to remove `private[connect]`, see also SPARK-16964 and SPARK-16813

Reply:
Makes sense at least for this dsl package. Removed `private[connect]` in this file.